Library/Homebrew/cask/audit.rb

```diff
@@ -270,6 +271,13 @@ def bad_osdn_url?
       bad_url_format?(/osd/, [%r{\Ahttps?://([^/]+.)?dl\.osdn\.jp/}])
     end
 
+    def check_url_verification
+      return unless cask.url.verified
+      return if cask.url.to_s.start_with?(cask.url.verified)
+
+      add_error "url #{cask.url} does not start with #{cask.url.verified}"
```

Comment:
It would be useful to add a [link to the documentation](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment), so the user knows how to proceed.

Technically the documentation URL will change once this is is accepted, but it’s better to have something in place now.

Also, since this replaces the `style` check, it should be removed in this PR as well (fine if it’s another commit).

Reply:
You can use https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-verified as the documentation link. I’ve made a [documentation PR](https://github.com/Homebrew/homebrew-cask/pull/84607) for when this is one is ready.